### ISSUE TITLE
YouTube entries cannot be accurately captioned, YouTube captions appear but are covered by player controls, and poor-quality automatic YouTube captions cannot be disabled in player

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#30]
Submitted in Kaltura Support Case #00554128:

> I believe the multiple issues with Kaltura ASR and YouTube entries have already been reported by folks from other universities, including UC Berkeley, University of New Mexico, and University of Minnesota. We are now encountering these issues in our production environment as well, following our migration to Kaltura ASR over the weekend.
> 
> First, YouTube entries can no longer be captioned by requesting captions through REACH. If you try to request captions for a YouTube entry, it looks like it is processing the captions request, but eventually fails. If the owner/editor of the media entry goes back to the captioning request page, it shows a "Status" of "Error," but does not explain the cause of the failure. This is a problem because the primary use case for putting YouTube entries in Kaltura was to provide accurate captions.
> 
> Second, YouTube entries are displaying the YouTube captions (in many cases, poor quality auto-captions) on the player underneath the player controls. The controls covering the captions' text might be considered a failure of [WCAG Success Criterion 1.4.12: Text Spacing (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html).
> 
> Third, the YouTube captions cannot be turned off, so they constantly cover up a portion of the video. This is especially problematic when dealing with poor quality auto-captions.
> 
> Please fix it so we can request accurate captions again for YouTube entries and not have poor quality auto-captions from YouTube displayed at all times.

**Comment by Rainribbon** [comment]
I received this response:

> At this time, Kaltura ASR does not support YouTube captioning. This limitation is primarily due to YouTube's policies, which are impacting all caption providers, not just Kaltura. That said, we are actively working with our Legal and Product teams to explore possible solutions.
